codec-http2/src/test/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionTest.java

```diff
@@ -696,11 +697,6 @@ public void weightChangeWithNoTreeChangeShouldNotifyListeners() throws Http2Exce
                 any(Http2Stream.class));
         verify(clientListener, never()).onPriorityTreeParentChanged(any(Http2Stream.class),
                 any(Http2Stream.class));
-        assertEquals(5, client.connectionStream().prioritizableForTree());
```

Comment:
@nmittler - the testing that was removed was to support the `prioritizableForTree` feature which was removed in this PR. I don't think this feature is necessary and it would complicate the task of removing all streams while iterating over the stream map.